packages/mendel-pipeline/src/cache/entry.js

```diff
@@ -5,6 +5,7 @@ class Entry {
         // property is filled by cache where they have more context
         this.normalizedId;
         this.variation;
+        this.runtime; // one of 'broser' or 'server'
```

Comment:
or `package` or `isomorphic`.

I thought about this. At the manifest walking time, we need to pick the right one, right?

I will write down in my note that this is a limitation with Mendel: same file cannot be pointed from multiple package.json. A file cannot be both isomorphic and browser/main.